src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java

```diff
@@ -810,7 +810,7 @@ public static void removeCommonFrames(final List<String> causeFrames, final List
      * @param <T> the type of Throwable you are searching.
      * @param throwable  the throwable to inspect, may be null
      * @param clazz  the class to search for, subclasses do not match, null returns null
-     * @return the index into the throwable chain, null if no match or null input
+     * @return the first matching throwable form the throwable chain, null if no match or null input
```

Comment:
s/form/from?

Reply:
Hi @kinow,
fixed the typo. Thanks for spotting it :-)